pep0/pep.py

```diff
@@ -62,34 +62,38 @@ class Author(object):
 
     def __init__(self, author_and_email_tuple):
         """Parse the name and email address of an author."""
+        self.first = self.last = ''
+
         name, email = author_and_email_tuple
         self.first_last = name.strip()
         self.email = email.lower()
-        last_name_fragment, suffix = self._last_name(name)
-        name_sep = name.index(last_name_fragment)
-        self.first = name[:name_sep].rstrip()
-        self.last = last_name_fragment
-        if self.last[1] == u'.':
-            # Add an escape to avoid docutils turning `v.` into `22.`.
-            self.last = u'\\' + self.last
-        self.suffix = suffix
-        if not self.first:
-            self.last_first = self.last
+
+        name_dict = self._parse_name(name)
+        self.suffix = name_dict.get("suffix")
+        if name_dict.get("name"):
```

Comment:
```suggestion
        if "name" in name_dict:
```

(avoid retrieving and discarding the value)